apps/avatax/src/modules/saleor/order.ts

```diff
@@ -47,3 +51,69 @@ export class SaleorOrder {
     return this.data.order.channel.taxConfiguration.pricesEnteredWithTax;
   }
 }
+
+interface ISaleorCancelledOrderPayload {
```

Comment:
You improved class name but interface name don't. Lets say Payload is something we keep for auto-generated types from graphql.

Then, we transform payload to our domain objects

So I suggest `SaleorCancelledOrderEvent` that implements `ISaleorCancelledOrderEvent`. Not perfect but should be good for now